db/migrate/20221030224657_add_action_fields_to_reports.rb

```diff
@@ -0,0 +1,20 @@
+class AddActionFieldsToReports < ActiveRecord::Migration[6.1]
+
+  def change
+    change_table :reports, bulk: true do |t|
+      t.integer :reported_author_id, index: true
+      t.string :action
+    end
+
+    Report.find_each do |report|
+      # get reported author id from item before item gets deleted
+      if report.reported_author.present?
+        report.reported_author_id = report.reported_author.id
+      end
+      if report.item.present?
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.